channels-rke2.yaml

```diff
@@ -3634,6 +3634,58 @@ releases:
         version: 3.5.0-rancher100
     serverArgs: &serverArgsv1-31-13-rke2r1
       <<: *serverArgsv1-31-12-rke2r1
+      helm-job-image:
+        type: string
+      disable-scheduler:
+        type: bool
+      disable-cloud-controller:
+        type: bool
+      servicelb-namespace:
+        type: string
+      enable-servicelb:
+        type: boolean
     agentArgs: &agentArgsv1-31-13-rke2r1
       <<: *agentArgsv1-31-12-rke2r1
+      private-registry:
+        type: string
+      node-ip:
+        type: string
+      node-external-ip:
+        type: string
+      container-runtime-endpoint:
+        type: string
+      kubelet-path:
+        type: string
+      node-name:
+        type: string
+      node-label:
+        type: string
+      node-taint:
+        type: string
+      cloud-controller-manager-image:
+        type: string
+      lb-server-port:
+        type: string
+      image-credential-provider-bin-dir:
+        type: string
+      image-credential-provider-config:
+        type: string
+      kube-proxy-image:
+        type: string
+      etcd-image:
```

Comment:
This one should be ok, but it's already in the server args. I know it's also available in agent args, but does it have any effect there? If not, we could remove it.